### PR TITLE
SSE-2829  frontend template  correct field name

### PR DESCRIPTION
**Pull request by abdul-gds**

### CODE REVIEW

**Review by MartinH-DI — DISMISSED** [review]

**Review by rafal-surowiec-co — CHANGES_REQUESTED** [review]
Do we need closing bracket ?

File: infrastructure/frontend/frontend.template.yml [hunk 1]
```diff
@@ -270,7 +270,7 @@ Resources:
       Type: A
       Name: !Sub
         - ${Name}${Domain}
-        - Name: !If [ Subdomain, !Sub "${DeploymentName}.", !Ref AWS::NoValue ]
+        - Name: !If [ Subdomain, !Sub "${DeploymentName}.", ""
```
Comment (MartinH-DI) [inline infrastructure/frontend/frontend.template.yml:273]: Its it: ""]

Comment (rafal-surowiec-co) [inline infrastructure/frontend/frontend.template.yml:273]: Do we need closing bracket ?

Comment (abdul-gds) [reply infrastructure/frontend/frontend.template.yml]: thanks for the review. corrected now

**Review by rafal-surowiec-co — APPROVED** [review]

**Pull request merged**